app/main.py

```diff
@@ -16,4 +16,32 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {}
+
+
+def mutable_immutable(*args) -> dict:
+    mutable = []
+    immutable = []
+    for element in args:
+        if isinstance(element, (list, set, dict)):
+            mutable.append(element)
+        else:
+            immutable.append(element)
+
+    sorted_variables["mutable"] = mutable
+    sorted_variables["immutable"] = immutable
```

Comment:
Modifying the global variable `sorted_variables` directly within the function can lead to unexpected behavior if the function is called multiple times. It's better to create a local dictionary within the function and return it.